graviti/manager/dataset.py

```diff
@@ -100,6 +108,37 @@ def __init__(
     def _repr_head(self) -> str:
         return f'{self.__class__.__name__}("{self.owner}/{self.name}")'
 
+    def _factory_getter(self, offset: int, limit: int, sheet_name: str) -> Dict[str, Any]:
```

Comment:
the name `factory_getter` should return a `factory`
also a normal method name should starts with a verb